README.md

```diff
@@ -350,27 +350,13 @@ $ make undeploy
 
 ## Metrics
 
-The LVM Operator runs a metrics exporter sidecar to export Prometheus metrics. To enable monitoring on OpenShift clusters, assign the `openshift.io/cluster-monitoring` label to the same namespace that you deployed LVMS to.
+To enable monitoring on OpenShift clusters, assign the `openshift.io/cluster-monitoring` label to the same namespace that you deployed LVMS to.
```

Comment:
what happens when the cluster monitoring is disabled? will we still have the certificates setup correctly, but just the scraping disabled?

Reply:
The certificate setup is working independently of enabling monitoring, so everything works but just the scraping gets disabled.